projects/ngx-formentry/src/components/machine-learning-button/model-helper.ts

```diff
@@ -210,7 +210,8 @@ export function generatePredictionPayload(
     } else if (
       keyPopulationVal == 5622 ||
       keyPopulationVal == 162277 ||
-      keyPopulationVal == 165100
+      keyPopulationVal == 165100 ||
+      keyPopulationVal === "bd370cad-06fe-4950-a36f-ed991b280ce6"
```

Comment:
does this still point to the same concept ID? Let's just keep one for 3.x.

Reply:
Ok